### PR TITLE
fix: use child process PID instead of parent PID for default name, and resolve issues with custom name and output destination

### PR DESCRIPTION
**Pull request by rubenpad**
- Fix issue [clinicjs/node-clinic#452](https://github.com/clinicjs/node-clinic/issues/452)
- Fix issue [clinicjs/node-clinic#450](https://github.com/clinicjs/node-clinic/issues/450)
- Fix usage of  the `--name` option to set a custom file name
- Fix usage of the `--name` and `--dest` options when they are passed together

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,10 @@ const heapProfiler = new ClinicHeapProfiler()
     **Default**: false
   - debug [`<boolean>`][] If set to true, the generated html will not be minified.
     **Default**: false
-  - dest [`<String>`][] The file where the collected data is stored.
-    **Default**: `./.clinic/<process.pid>.clinic-heapprofile`
+  - dest [`<String>`][] Destination for the collected data
+    **Default**: `.clinic`
+  - name File name for the collected data
```
Comment (RafaelGSS) [inline README.md:56]:
```suggestion
  - name [`<String>`][] File name for the collected data
```

Comment (rubenpad) [reply README.md]: Done. Thanks!

**Review by RafaelGSS — APPROVED** [review]

**Comment by RafaelGSS** [comment]
FYI we need to make this change in other tools (flame, bubbleprof and doctor) to enable it via `clinic [TOOL]`

**Comment by rubenpad** [comment]
> FYI we need to make this change in other tools (flame, bubbleprof and doctor) to enable it via `clinic [TOOL]`

I think this already works in the other tools. I have executed the following commands using the latest version of clinic and they work well:

- `clinic [TOOL] --dest my-custom-dest -- node my-script.js`
- `clinic [TOOL] --name my-custom-name -- node my-script.js`
- `clinic [TOOL] --dest my-custom-dest --name my-custom-name -- node my-script.js`

**Comment by RafaelGSS** [comment]
Great, I forgot I did it already.

**Pull request merged**